dyninstAPI/src/image.C

```diff
@@ -1049,11 +1037,7 @@ pdmodule *image::findModule(const string &name, bool wildcard)
       }
    }
 
-   //cerr << " (image::findModule) did not find module, returning NULL" << endl;
    if (found) {
-      // Just-in-time...
-      //if (parseState_ == symtab)
-      //analyzeImage();
       return found;
    }
 
```

Comment:
replace this and the `return NULL;` with `return found;`

Reply:
Fixed. Thanks for the careful look!